ui/raidboss/browser_tts_engine.ts

```diff
@@ -21,28 +25,44 @@ class SpeechTTSItem extends TTSItem {
   }
 }
 
-class GoogleTTSItem extends TTSItem {
-  constructor(text, lang) {
-    super();
-    this.lang = lang;
+class GoogleTTSItem implements TTSItem {
+  readonly text: string;
+  readonly lang: string;
+  private item?: Element | null;
```

Comment:
(just for my own knowledge, does the `?` already imply can be null, or do you also have to say null?)

Reply:
Or, looking at this again, when can it be null? Should it just be `private item?: Element`?

Reply:
`?` implies `undefined`; in this case, `Document.createElement` returns `HTMLIFrameElement | null`. Removing the `| null` will return:
```
Type 'Element | null' is not assignable to type 'HTMLIFrameElement | undefined'.
  Type 'null' is not assignable to type 'HTMLIFrameElement | undefined'
  ```